src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.ReinferredVariableTracker.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal sealed partial class NullableWalker
+    {
+#nullable enable
+        internal sealed class ReinferredVariableTracker
+        {
+            internal readonly struct Immutable
+            {
+                private readonly ReinferredVariableTracker? _tracker;
```

Comment:
>ReinferredVariableTracker? _tracker [](start = 33, length = 35)

Is this immutable? What prevents another call to `SetIReinferredSymbol`?

Reply:
I could call the struct `Readonly` instead? The intent is much like `ReadOnlySpan`

---
In reply to: [327278382](https://github.com/dotnet/roslyn/pull/38769#discussion_r327278382) [](ancestors = 327278382)